src/devices/Bmx280/Bme280.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Device.I2c;
+
+namespace Iot.Device.Bmx280
+{
+    public class Bme280 : BmxBase
+    {
+        private const byte Signature = 0x60;
+        public const byte Address = 0x76;
+
+        public Bme280(I2cDevice i2cDevice)
```

Comment:
Do we plan to add SPI support for these devices in this PR? Or do we want to do that in a future PR instead?

Reply:
i think we should add it once someone requests it - also I expect we will eventually have some common interface between those 2

Reply:
We Don't have this sensor to test in SPI mode. So yes, I'll leave it up to the point we either find one to test, either someone demands it. Looks like most popular implémentations as using I2C.